apps/products-feed/src/public/vercel.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="283" height="64" viewBox="0 0 283 64" fill="none" 
```

Comment:
Unused file, can be removed

Reply:
actualy removing entire "public" dir, its also not correct (it can't be in src)